go/pkg/cas/upload.go

```diff
@@ -163,26 +172,43 @@ func (u *uploader) startProcessing(ctx context.Context, in *UploadInput) error {
 		// Do not use os.Stat() here. We want to know if it is a symlink.
 		info, err := os.Lstat(absPath)
 		if err != nil {
-			return errors.Wrapf(err, "lstat failed")
```

Comment:
why remove this wrap?

Reply:
turns out lstat errors start with "lstat: ", so this didn't add much.

Reply:
I think stacktrace is helpful when showing this error?

Reply:
+1, trace is useful (or even more verbose wording to make it easier to find, eg "error while stating uploadinput")

Reply:
Added `errors.WithStack`

Reply:
seems not added?

Reply:
ah, I've added it to a different Lstat call. Now both have WithStac